src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -586,7 +594,15 @@ where
         )?;
         ro_circuit.absorb_base(WrapValue::Assigned(assigned_public_params_hash.clone()));
 
-        let assigned_W = assign_and_absorb_point!(self.W)?;
+        let assigned_W: Result<Vec<AssignedPoint<C>>, Error> = self
+            .W
+            .iter()
+            .map(|W| -> Result<AssignedPoint<C>, Error> {
+                let res = assign_and_absorb_point!(W)?;
+                Ok(res)
+            })
+            .collect();
+        let assigned_W = assigned_W?;
```

Comment:
var not needed here
```rust
.collect()?;
```

Reply:
I tried this before but got error, so I use above work around.

Error: `Try` is not implemented for `Vec<Result<ecc::AssignedPoint<C>,Error>`
Any idea how to fix it?